ng2/5-piratenameservice/lib/src/name_service.dart

```diff
@@ -12,16 +12,31 @@ class NameService {
   static final Random _indexGen = new Random();
 
   final _names = <String>[
-    'Anne', 'Mary', 'Jack', 'Morgan', 'Roger',
-    'Bill', 'Ragnar', 'Ed', 'John', 'Jane'
+    'Anne',
+    'Mary',
+    'Jack',
+    'Morgan',
+    'Roger',
+    'Bill',
+    'Ragnar',
+    'Ed',
+    'John',
+    'Jane',
   ];
   final _appellations = <String>[
-    'Jackal', 'King', 'Red', 'Stalwart', 'Axe',
-    'Young', 'Brave', 'Eager', 'Wily', 'Zesty'
+    'Jackal',
+    'King',
+    'Red',
+    'Stalwart',
+    'Axe',
+    'Young',
+    'Brave',
+    'Eager',
+    'Wily',
+    'Zesty',
   ];
 
-  String _randomFirstName() =>
-      _names[_indexGen.nextInt(_names.length)];
+  String _randomFirstName() => _names[_indexGen.nextInt(_names.length)];
```

Comment:
(And until we find/develop a scheme that allows us to change the dartfmt defaults on a per-file or per-folder basis, I suggest that we stick to the defaults.)